tox.ini

```diff
@@ -9,7 +9,7 @@ commands =
 extras = md
 
 [testenv:pep8]
-basepython = python3.6
+basepython = python3
```

Comment:
The CI was launched using [python 3.x](https://github.com/pypa/readme_renderer/blob/main/.github/workflows/ci.yml#L30) but this tox target wanted 3.6 specifically, so it couldn't work. Don't know why it did so far. If you want this in a different PR, let me know.